.github/workflows/release_dev.yml

```diff
@@ -3,7 +3,7 @@ on: [ workflow_dispatch ]
 
 jobs:
   publish:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v4
         with:
```

Comment:
No `advzip` install for these the rest of the workflows?

Reply:
given that it's not used in these release channels, I opted not to include it to save CI time. if you want me to add it then I can

Reply:
`dev` and `pre` are both configured to use `LIBDEFLATE` compression, which requires `advzip` from what I'm seeing, so probably best to install it for these workflows too.

Reply:
ah I forgor 💀, will do now